tests/bdd/features/snapshot/list/test_list.py

```diff
@@ -0,0 +1,116 @@
+"""Volume Snapshot listing feature tests."""
+
+import pytest
+from pytest_bdd import given, scenario, then, when, parsers
+
+import openapi.exceptions
+from common.apiclient import ApiClient
+from common.deployer import Deployer
+from openapi.model.create_pool_body import CreatePoolBody
+from openapi.model.create_volume_body import CreateVolumeBody
+from openapi.model.protocol import Protocol
+from openapi.model.publish_volume_body import PublishVolumeBody
+from openapi.model.volume_policy import VolumePolicy
+
+VOLUME1_UUID = "d01b8bfb-0116-47b0-a03a-447fcbdc0e99"
+POOL1_NAME = "pool-1"
+NODE1 = "io-engine-1"
+VOLUME1_SIZE = 1024 * 1024 * 32
+VOLUME1_SNAP_IDS = [
+    "1f49d30d-a446-4b40-b3f6-f439345f1ce9",
+    "2f49d30d-a446-4b40-b3f6-f439345f1ce9",
+    "3f49d30d-a446-4b40-b3f6-f439345f1ce9",
+    "4f49d30d-a446-4b40-b3f6-f439345f1ce9",
+]
+
+
+@scenario("list.feature", "Volume snapshots list")
+def test_volume_snapshots_list():
+    """Volume snapshots list."""
+
+
+@given("a deployer cluster")
+def a_deployer_cluster():
```

Comment:
let's mark this as a module fixture (might need to split impl into separate fixture perhaps) to avoid recreating cluster for every example